app/assets/stylesheets/prism.scss

```diff
@@ -0,0 +1,159 @@
+
```

Comment:
This is a specific version that Ed Horsford created. Although don't know what the difference is between this and the CDN version https://github.com/alphagov/govuk_elements/blob/69070e62f3fdffc46b67dec16557913e0af599cb/assets/sass/vendor/prism.scss